### DISCUSSION TITLE
미팅 참가 여부 확인 API 개선

### DISCUSSION THREAD

**Discussion by waterfogSW** [Student-Center/weave-server#233] [개발 논의]
- 클라이언트측에서 미팅 참가여부를 조회하는 API를 사용할때 상대팀, 내팀 구별이 안되서 추가적인 로직처리가 이루어지고 있음. 내팀, 상대팀 구별 필요.
- 내가 참가 의사 신청을 했는지 여부를 내려주면 버튼 비활성화가 가능해 사전 표시가 가능할것

**Comment by dojinyou** [comment]
멤버 ID 받고 해당 멤버가 요청자의 UserID를 가지는지 체크하고 참가 신청했는지 체크하시는 식으로 할까요?

**Reply by waterfogSW** [reply]
멤버아이디를 클라이언트로부터받는다는 말씀이실까용, 이미 토큰에 사용자 ID있어서 받는건 그대로 해도 되지 않나요?
